packages/dashmate/src/templates/writeServiceConfigsFactory.js

```diff
@@ -20,7 +20,9 @@ function writeServiceConfigsFactory() {
     // Drop all files from configs directory
     const configDir = path.join(HOME_DIR_PATH, configName);
 
-    fs.rmSync(configDir, { recursive: true, force: true });
+    if (!fs.existsSync(configDir)) {
```

Comment:
Won't we create a directory bellow?

Reply:
Yes. The logic was to purge the directory before render. I disabled removal and used check on folder existence instead

Reply:
I meant line 30-32:

```js
      // Recreate it
      fs.mkdirSync(absoluteFileDir, { recursive: true });
```